### PR TITLE
Add AudioIDs

### PR DESCRIPTION
**Pull request by oliver-koch**
yaml/11000289.yaml (PAW Patrol - Hier kommt Tracker)
yaml/11000201.yaml (Die Sendung mit dem Elefanten - Schlaf schön)
yaml/11000248.yaml (PAW Patrol-Zuma)

### CODE REVIEW

File: yaml/11000248.yaml
Comment (SciLor) [inline yaml/11000248.yaml:1]: Why did you remove the audio id? This is a previously valid audio id. Please add a new valid entry as first entry and keep the old one.

Comment (oliver-koch) [reply yaml/11000248.yaml:1]: removed was a copy paste error done by my first PR. the now provided is the correct one. sorry for that.

Comment (oliver-koch) [reply yaml/11000248.yaml:1]: 1699434320 = "model": "11000249",
                         "title": "Disney - Tiggers großes Abenteuer",
                         "series": "Disney",
